src/main/webapp/app/overview/course-conversations/dialogs/channels-overview-dialog/channels-overview-dialog.component.ts

```diff
@@ -136,7 +139,7 @@ export class ChannelsOverviewDialogComponent extends AbstractDialogComponent imp
             )
             .subscribe({
                 next: (channels: ChannelDTO[]) => {
-                    this.channels = channels ?? [];
+                    this.channels = channels ? channels.filter((channel) => channel.subType === this.channelSubType) : [];
```

Comment:
imo this is cleaner but this might be personal preference.
```suggestion
                    this.channels = channels?.filter((channel) => channel.subType === this.channelSubType) ?? [];
```